src/services/apis/auth.ts

```diff
@@ -103,16 +103,22 @@ const jsonHeader = {
 }
 
 const baseQuery = fetchBaseQuery({ 
-  baseUrl: api_url,
+  baseUrl: apiUrl,
   prepareHeaders: (headers) => {
-    const token = ls_storage.get(ls_user_keys.accessToken)
+    const token = lsStorage.get(lsUserKeys.accessToken)
     if (token) {
       headers.set('Authorization', `Bearer ${token}`);
     }
     return headers;
   },
 })
 
+const protectedApiRoutes:string[] = [
```

Comment:
refactor api: refresh token method will be applied only on route in protectedApiRoutes list